package.json

```diff
@@ -91,6 +92,7 @@
     "eslint-plugin-react": "7.11.1",
     "file-loader": "2.0.0",
     "flow-bin": "0.86.0",
+    "flow-coverage-report": "^0.6.1",
```

Comment:
I would continue with the current philosophy of specifying the version and not leave it open.